Jenkinsfile

```diff
@@ -327,6 +327,13 @@ def runNixTest(prefix) {
             sh './simplefs.test -test.timeout 2m'
         }
     }
+    tests[prefix+'kbfsgit'] = {
+        dir('kbfsgit') {
+            sh 'go test -i'
+            sh 'go test -race -c'
+            sh './kbfsgit.test -test.timeout 10m'
```

Comment:
This command doesn't build it, but all binaries are built with a `go install` command above.